scss/civicrm/contact/pages/_import-contact.scss

```diff
@@ -94,6 +94,11 @@
       padding: 20px 0 0 0 !important;
     }
 
+    #wizard-steps {
```

Comment:
Why there are 2 `#wizard-steps` in file?

Reply:
The first one is for all the import contact pages - i.e. all the four steps. The second one is specifically for the Preview step, which is styled in a different way (i.e. there is no padding on the crm-block div).

Reply:
@mihaelmladenov can you add all screenshots?

Reply:
@mukeshcompucorp I have added screenshots of all affected pages.

Reply:
why does it need to be nested inside the first `#wizard-steps` then? can't it be
```scss
#wizard-steps {

}

#Preview {

  #wizard-steps {

  }
}
```
this would make the selector much less confusing

Reply:
The import wizard has several steps. One of the steps is Previewing. On this step we have a different HTML than the other ones, which means that we need a different left and right margin than the others. That's why I have done it this way.